rules/dates-do-you-keep-date-formats-consistent-across-your-application/rule.md

```diff
@@ -44,10 +44,23 @@ Figure: Bad Example - Using hard coded formatting string
 'Initial CultureInfo settings for the application
 Public initialCulture As CultureInfo
 ...
+
 ...
 txtDateCreate.Text = CType(txtDateCreate.Tag, System.DateTime).ToString(initialCulture.DateTimeFormat)
 ```
 
 ::: good
 Figure: Good Example - Using culture info to format datetime
 :::
+
+## Browser language settings
+
+It is required to set web browser language to English (Australia) after modifying the computer reginal and language settings. This ensures proper formatting and compatibility, particularly when downloading web pages or accessing specific web content. 
```

Comment:
Hi @ClaraFang-SSW 

- [ ] 1. We should have a link to https://www.ssw.com.au/rules/make-sure-all-software-uses-english/ somewhere here. Please add